Packs/Gurucul/ReleaseNotes/1_2_0.json

```diff
@@ -0,0 +1 @@
+{"breakingChanges":true,"breakingChangesNotes":null}
```

Comment:
This file should be removed because there is no breaking changes

Reply:
Removed. It was added earlier because it was breaking backward compatibility. Now that is removed, this is not required.